### DISCUSSION TITLE
400GB for llama 3?!?! should we be making our own gpu now?

### DISCUSSION THREAD

**Discussion by ouvaa** [ggml-org/llama.cpp#6749] [Q&A]
can someone take a guess what's the most powerful amd epyc 9000 series and motherboard and ddr ram types and what kind of t/s we can expect to get to load a 400GB llama 3 in future?

**Comment by phymbert** [comment]
Meta casually trains it on 16k GPUs, so we have a limit :)

**Reply by ouvaa** [reply]
let's say they put out a 1TB in future, what do u expect non multimillionaires to do about that? live with 0.1 t/s?

that's why i created this language. i think it's important normal people have access to AI usage at scale
https://github.com/ouvaa/ouvaa.github.io

**Comment by mirek190** [comment]
> can someone take a guess what's the most powerful amd epyc 9000 series and motherboard and ddr ram types and what kind of t/s we can expect to get to load a 400GB llama 3 in future?

So even if you use 4b version you still need 512 GB RAM ...

**Reply by ouvaa** [reply]
@mirek190 i dont think i can use anything lesser than 281GB after we have chatgpt4. anything else is really too dumb for me. either a self hosted chatgpt4 close to equivalent or nothing.

**Reply by mirek190** [reply]
Have you tested llama 3 8b?
That model is insane ...in some cases I'd a good as gpt-4

**Reply by anianios** [reply]
@mirek190 , can it run on llama.cpp? I tried to run it yesterday using the .server and it was hanging. Have you been successful in doing it?

**Reply by supportend** [reply]
I use the instruct models (8b and 70b), both are working with main on my system. I'm not sure, i use the correct prompt format, but it works.

**Reply by mirek190** [reply]
I  am testing from main.exe of llamacpp

main.exe --model models/new3/Meta-Llama-3-8B-Instruct.Q8_0.gguf --color --threads 30 --keep -1 --n-predict -1 --repeat-penalty 1.1 --ctx-size 0 --interactive -ins -ngl 99 --simple-io -r '<|eot_id|>' --in-prefix "\n<|start_header_id|>user<|end_header_id|>\n\n" --in-suffix "<|eot_id|><|start_header_id|>assistant<|end_header_id|>\n\n" -p "<|begin_of_text|><|start_header_id|>system<|end_header_id|>\n\nYou are a helpful, smart, kind, and efficient AI assistant. You always fulfill the user's requests to the best of your ability.<|eot_id|>\n\n"

**Reply by anianios** [reply]
Has anyone been able to make it work with server process instead of the main?

**Reply by ouvaa** [reply]
@mirek190 i've spent a lot of time playing with llama before but seriously, i dont think it's worth touching anything that's 22b and below... but since u say so, i'll try someday. i'm more looking forward to 281GB Mixtral now. is the llm ranking for both llama 3 and mixtral 281 out yet?

**Reply by mirek190** [reply]
From leaderboard lmsys  llama 3 8b has level of mistal 8x22b what is insane.
